### PR TITLE
Added zulip data to html

### PR DESCRIPTION
**Pull request by AbhayGoyal**

### CODE REVIEW

File: _includes/2017_data/sponsors.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<!-- Clients Aside -->
+i<!-- Clients Aside -->
```
Comment (OriolAbril) [inline _includes/2017_data/sponsors.html]:
```suggestion
<!-- Clients Aside -->
```

File: _includes/2017_data/sponsors.html [hunk 2]
```diff
@@ -12,6 +12,14 @@ <h2 class="section-heading" id="sponsors">Sponsors</h2>
...
                 </div>
             </div>
+
+            <div class = "row">
+	    	<div class = "col-md-4 col-sm-6">
+			<a href = "https://zulip.com/" class = "portfolio-link" data-toggle = "modal">
+			<img src = "./css/2017_style/img/sponsors/zulip.png" class = "image-responsive img-centered" alt = "zulip">
+			</a>
+		</div>
+	    </div>
```
Comment (OriolAbril) [inline _includes/2017_data/sponsors.html]: Not sure if you are able to preview the website locally, but this looks like it's adding the zulip logo beside the tidelift one, in the "metropolis hastings" tier

What should happen is adding a new section with a header like the metropolis hasting tier one below it, the new header should say "Technology partners" and have the zulip logo under this new header

**Review by OriolAbril — CHANGES_REQUESTED** [review]

File: _includes/2017_data/sponsors.html [hunk 3]
```diff
@@ -12,6 +12,14 @@ <h2 class="section-heading" id="sponsors">Sponsors</h2>
                     <h3 class="section-heading">Metropolis-Hastings Tier</h3>
```
Comment (canyon289) [inline _includes/2017_data/sponsors.html:15]: Would add another section below the tidelift image for "technology partners" and then move the added code below

File: _includes/2017_data/sponsors.html (hunk 2 above)
Comment (AbhayGoyal) [reply _includes/2017_data/sponsors.html]: Oh ok...understood, will change that also. I am still working out of vi, don't know any other way to edit the files. Is there a better way than vi? emacs somehow does not work

**Comment by canyon289** [comment]
Looking almost good. NumFOCUS image link is broken and centering is off for some reason

File: _includes/2017_data/sponsors.html [hunk 4]
```diff
@@ -1,37 +1,65 @@
...
+                <div class = "col-md-12 text-center">
+                    <h3 class = "section-heading">NumFOCUS</h3>
+                </div>
+            </div>
+            <div class="row">
                 <div class="col-md-4 col-sm-6">
-                    <a href="#" class="portfolio-link" data-toggle="modal">
-                        <img src="./css/2017_style/img/logos/TBD.jpg" class="img-responsive img-centered" alt="">
+                    <a href="https://numfocus.org/" class="portfolio-link" data-toggle="modal">
+                        <img src="./css/2017_style/img/sponsors/numfocus-logo.png" class="img-responsive img-centered" alt="numfocus">
```
Comment (canyon289) [inline _includes/2017_data/sponsors.html]: This should be .jpg

Comment (AbhayGoyal) [reply _includes/2017_data/sponsors.html]: Changed to jpg

File: _includes/2017_data/sponsors.html [hunk 5]
```diff
@@ -1,45 +1,65 @@
...
+                <div class = "col-lg-12 text-center">
+                    <h2 class = "section-heading" id = "techpartner">Technology Partners</h2>
+                </div>
+            </div>
+
+            <div class = "row">
+                <div class = "col-md-12 text-center">
+                    <h3 class = "section-heading">Zulip</h3>
+                </div>
+            </div>
```
Comment (OriolAbril) [inline _includes/2017_data/sponsors.html:37]: I think it's not necessary to add a header for each company, the logo is enough

**Review by OriolAbril — APPROVED** [review]

File: _includes/2017_data/sponsors.html (hunk 5 above)
Comment (AbhayGoyal) [reply _includes/2017_data/sponsors.html:37]: Ok, so should I remove the headings then?

Comment (OriolAbril) [reply _includes/2017_data/sponsors.html:37]: not the headings, only "zulip" as heading, the tier heading are already right

**Comment by canyon289** [comment]
Thanks @AbhayGoyal

**Pull request merged**